netdiff/parsers/base.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, data=None, url=None, file=None,
         :param metric: routing protocol metric
         :param timeout: timeout in seconds for HTTP or telnet requests
         :param verify: boolean (valid for HTTPS requests only)
+        :param directed: whether the resulting graph should be directed (undirected by default for historical compatibility)
```

Comment:
I'd change `historical` with `backward` and wrap the line before the rounded parentheses, eg:
```
:param directed: whether the resulting graph should be directed
                 (undirected by default for backward compatibility)
```